proptest/src/test_runner/config.rs

```diff
@@ -80,8 +78,6 @@ pub fn contextualize_config(mut result: Config) -> Config {
         }
     }
 
-    result.failure_persistence =
-        Some(Box::new(FileFailurePersistence::default()));
```

Comment:
seems like this was in some sense the default for the `std` feature? does it make sense adjust `DEFAULT_CONFIG` to be setup with `Some(Box::new(FileFailurePersistence::default()))` if feature is `std`?

Reply:
fair point. will update the `Default` impl